Client/Frontend/Browser/HomePanel/NewTabPageBackgroundDataSource.swift

```diff
@@ -29,7 +29,7 @@ class NewTabPageBackgroundDataSource {
         }
         
         lazy var imageLiteral: UIImage? = {
-            return UIImage(named: imageUrl)
+            return UIImage(contentsOfFile: imageUrl)
```

Comment:
Leaving this named should work, since all background are bundled.
And use `contentsOfFile` only for sponsored images at line 47